doc/Client-Portal Protocol.md

```diff
@@ -514,14 +532,14 @@ This a format for client reconnects.
 
   object(SOACMessage)::
     {
-     id: string(SessionId), /* StreamId returned in publishing or SubscriptionId returned in subscribing*/
-     signaling: object(OfferAnswer) | object(CandidateMessage) | object(RemovedCandidatesMessage)
+     id: string(transportId), /* Transport ID returned in publishing or subscribing*/
+     signaling: object(OfferAnswer) | object(CandidateMessage) | object(RemovedCandidatesMessage) | object(P2PQuicParametersMessage)
     }
 
     object(OfferAnswer)::
       {
        type: "offer" | "answer",
-       sdp: string(SDP)
+       sdp: string(SDP) | null,  // WebRTC connection
```

Comment:
null for "quic"?

Reply:
Sounds like it will not be `null` because `P2PQuicParametersMessage` is added.